qutip/solver/multitrajresult.py

```diff
@@ -337,13 +337,16 @@ def _target_tolerance_end(self):
             # We only include traj. without abs. weights in this calculation.
             # Since there are traj. with abs. weights., the weights don't add
             # up to one. We have to consider that as follows:
-            #   <(x - <x>)^2> / <1> = <x^2> / <1> - <x>^2 / <1>^2
+            # err = (std * <w>**2 / (N-1)) ** 0.5
+            # avg = <x * w>
+            # avg2 = <x**2 * w>
+            # std * <w>**2 = (<x**2> - <x>**2) * <w>**2
+            #              = avg2 * <w> - avg**2
             # and "<1>" is one minus the sum of all absolute weights
             one = one - self._total_abs_weight
 
-        target_ntraj = np.max((avg2 / one - (abs(avg) ** 2) / (one ** 2)) /
-                              target**2 + 1)
-
+        std = avg2 * one - abs(avg)**2
+        target_ntraj = np.max(std / target**2) + 1
```

Comment:
Thank you for fixing this calculation. I did my best to understand how the target tolerance calculation works, but I don't think I really understood it. (That is, even without weights.)

If there is a reference explaining the target tolerance calculation, it might be useful linking to it in the users guide.

In line 345, should that `"<1>"` be a `"<w>"` now?